integration/test/crmapping/crmapping_test.go

```diff
@@ -0,0 +1,135 @@
+package crmapping
+
+import (
+	"context"
+	"io/ioutil"
+	"path"
+	"strconv"
+	"testing"
+
+	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/giantswarm/operatorkit/resource"
+	"gopkg.in/yaml.v2"
```

Comment:
I've noticed that in most upstream projects they use `sigs.k8s.io/yaml` instead of `gopkg.in/yaml.v2`. Can we run into some edge cases where the behaviour would be different?

Reply:
I'm not aware of any. So far we have been always using `gopkg.in/yaml.v2` or `ghodss/yaml`. My quick grep over ~50 common repos found from my machine yields only few hits that are using k8s one and all rest using either one of aforementioned alternatives.

For consistency purposes I figure it would be a good thing to settle on one of these and change rest of them, but until anything breaks, it doesn't probably matter that much which one is used. While YAML itself is pretty complicated format with anchors & all, the used subset in k8s context is most of the time very simple so I don't expect to hit corner cases anytime soon. :slightly_smiling_face:

Reply:
Ok. I ended up having some struggles with YAML parsing in the end so among other changes I switched to k8s parser. Thanks for note! :+1: